plugins/pencilblue/controllers/actions/admin/sites/delete_site.js

```diff
@@ -30,12 +30,13 @@ module.exports = function DeleteSiteActionModule(pb) {
         var self = this;
         var siteid = self.pathVars.siteid;
         var siteQueryService = new pb.SiteQueryService();
+        var SiteService = new pb.SiteService();
```

Comment:
Please use variables that start with a lowercase character for instance variable names.  We typically reserver initial upper camel case for prototype names.